src/compiler/scala/tools/nsc/settings/MutableSettings.scala

```diff
@@ -455,6 +455,7 @@ class MutableSettings(val errorFn: String => Unit)
     type T = Boolean
     protected var v: Boolean = false
     override def value: Boolean = v
+    def boolValue: Boolean = v
```

Comment:
@som-snytt I think that the alternative below from @lrytz may avoid the boxing as well, and be a sumpler change as this should not affect user code - it could just change BooleanSetting etc

Will have a look when I can get other PRS unstalled